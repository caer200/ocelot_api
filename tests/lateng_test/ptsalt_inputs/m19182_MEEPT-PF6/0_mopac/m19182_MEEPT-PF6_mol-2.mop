CHARGES

 
P	10.492585 +1 1.122343 +1 14.914490 +1
F	8.995712 +1 1.405040 +1 14.408118 +1
F	11.968931 +1 0.799116 +1 15.405603 +1
F	10.082807 +1 1.543518 +1 16.396547 +1
F	10.881611 +1 0.696440 +1 13.416224 +1
F	10.051062 +1 -0.388413 +1 15.212119 +1
F	10.891258 +1 2.618238 +1 14.591959 +1
