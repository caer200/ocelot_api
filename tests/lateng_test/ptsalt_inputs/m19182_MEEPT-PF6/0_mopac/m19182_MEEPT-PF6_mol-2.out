 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:08:36 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       P         10.49258500  *   1.12234300  *  14.91449000  *
     2       F          8.99571200  *   1.40504000  *  14.40811800  *
     3       F         11.96893100  *   0.79911600  *  15.40560300  *
     4       F         10.08280700  *   1.54351800  *  16.39654700  *
     5       F         10.88161100  *   0.69644000  *  13.41622400  *
     6       F         10.05106200  *  -0.38841300  *  15.21211900  *
     7       F         10.89125800  *   2.61823800  *  14.59195900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         P         10.4926      1.1223     14.9145
     2         F          8.9957      1.4050     14.4081
     3         F         11.9689      0.7991     15.4056
     4         F         10.0828      1.5435     16.3965
     5         F         10.8816      0.6964     13.4162
     6         F         10.0511     -0.3884     15.2121
     7         F         10.8913      2.6182     14.5920

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: P F6  =     7 atoms



      MOLECULAR POINT GROUP   :   Ci  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          P      2      3      4      5      6      7
      2          F      1
      3          F      1
      4          F      1
      5          F      1
      6          F      1
      7          F      1

     Ion Atom No.  Type    Charge
 
    1       1      P       -1

          COMPUTED CHARGE ON SYSTEM: -1

          COMPUTED CHARGE ON SYSTEM:  -1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
